### PR TITLE
Update the changelog for 1.3.1

### PR DESCRIPTION
**Pull request by sunank200**
- Update the changelog for 1.3.1

### CODE REVIEW

File: python-sdk/docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## 1.3.1
+
+### Feature:
+- Introduce a parameter `if_exists` which user can set to `append` or `replace` for `aql.dataframe()` operator. `append` insert into an existing table (`output_table`) from a Dataframe. [#1379](https://github.com/astronomer/astro-sdk/issues/1379)
```
Comment (kaxil) [inline python-sdk/docs/CHANGELOG.md:6]:
```suggestion
- `aql.dataframe()` operator now supports `append` mode in addition to the default of `replace`. `append` inserts records into an existing table (`output_table`) from the dataframe. [#1379](https://github.com/astronomer/astro-sdk/issues/1379)
```

File: python-sdk/docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+- Temporary Schemas accounts for Location/Region when creating BigQuery schema. User can now create schema in Bigquery as per region like - `<dataset_id>__<region_id>`. [#1369](https://github.com/astronomer/astro-sdk/issues/1369)
```
Comment (kaxil) [inline python-sdk/docs/CHANGELOG.md:7]: Would have been good to add a quick example. Have we covered this case in docs too? I don't see anything in https://github.com/astronomer/astro-sdk/pull/1416/files . How would a user know about this feature?

cc @phanikumv @utkarsharma2

File: python-sdk/docs/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+
+### Bug fixes
+- Fix the `aql.cleanup()` operator as failing as the attribute `output` was implemented in 2.4.0 [#1359](https://github.com/astronomer/astro-sdk/issues/1359)
```
Comment (kaxil) [inline python-sdk/docs/CHANGELOG.md:10]: This comment does not give much info from a User's perspective as they don't know what `output` attribute is?

It should be the following instead:

```suggestion
- Make `aql.cleanup()` operator compatible with Task Mapping features introduced in Airflow 2.4.0 [#1359](https://github.com/astronomer/astro-sdk/issues/1359)
```

File: python-sdk/docs/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,24 @@
...
+- Fix the backward compatibility with `apache-airflow-providers-snowflake==4.0.2`. [#1351](https://github.com/astronomer/astro-sdk/issues/1351)
+- LoadFile operator returns a dataframe if not using XCom backend.[#1348](https://github.com/astronomer/astro-sdk/pull/1348),[#1337](https://github.com/astronomer/astro-sdk/issues/1337)
```
Comment (kaxil) [inline python-sdk/docs/CHANGELOG.md:12]: Some of these points end with a "dot" while the others don't, I am happy with either but let's be consistent

Comment (kaxil) [reply python-sdk/docs/CHANGELOG.md:12]:
```suggestion
- Fix backward compatibility with `apache-airflow-providers-snowflake==4.0.2`. [#1351](https://github.com/astronomer/astro-sdk/issues/1351)
- `LoadFile` operator returns a dataframe when not using XCom backend shipped with Astro SDK [#1348](https://github.com/astronomer/astro-sdk/pull/1348),[#1337](https://github.com/astronomer/astro-sdk/issues/1337)
```

**Comment by sunank200** [comment]
Closing this as @utkarsharma2 already merged it to main.

**Pull request closed without merging**